src/abs.lean

```diff
@@ -0,0 +1,27 @@
+/-
```

Comment:
I think this file should be in a subfolder since we don't have any other standalone files in "src/", though I'm not sure which one makes the most sense. Any other maintainers want to weigh in?

Reply:
When in doubt, I just put things in `data`. Or maybe `algebra` is not a terrible choice, since there are a couple of definitions of `abs`?

Reply:
I wasn't sure whether it should go in `algebra` or `analysis`. I've moved it to `algebra` for now.